subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/InstantExecutionState.kt

```diff
@@ -133,7 +134,7 @@ class InstantExecutionState(
         if (gradle.includedBuilds.isNotEmpty()) {
             logNotImplemented(
                 feature = "included builds",
-                documentationSection = "config_cache:not_yet_implemented:composite_builds"
+                documentationSection = DocumentationSections.compositeBuilds
```

Comment:
💄 It would be nice if the reference could convey that this is a "not yet implemented" section vs. a "requirements" one

Reply:
Done.